frontend/components/organisms/heroes/media-small.json

```diff
@@ -0,0 +1,30 @@
+
+{
+   "variants": [
+      {
+        "images": [ "${image-id}" ],
+        "title": "Cement Products Breadcrumbs",
```

Comment:
I'd suggest changing this dummy text to something more generic like Lorem Ipsum text.

Reply:
Yes, I'll update this